nix/project-package-list.nix

```diff
@@ -1 +1 @@
-[ "cardano-balance-tx" "cardano-coin-selection" "cardano-numeric" "cardano-wallet" "cardano-wallet-launcher" "cardano-wallet-primitive" "cardano-wallet-test-utils" "delta-store" "delta-table" "delta-types" "text-class" "wai-middleware-logging" ]
+[ "cardano-balance-tx" "cardano-coin-selection" "cardano-numeric" "cardano-wallet" "cardano-wallet-benchmarks" "cardano-wallet-launcher" "cardano-wallet-primitive" "cardano-wallet-test-utils" "delta-store" "delta-table" "delta-types" "text-class" "wai-middleware-logging" ]
```

Comment:
Hi @paolino 

Is this addition of `cardano-wallet-benchmarks` intentional?

Asking as it seems to be unrelated to the PR title and description.

Reply:
mhh, I think there was a request from @HeinrichApfelmus to run nix/regenerate.sh , or maybe I confused it from the previous PR. I didn't edit this by hand if this is the question.

Reply:
Moved to a new commit. On a  tangent note, CI doesn't require we run nix/regenerate.sh but I think it should as it's doing it before running its steps.

Reply:
> Is this addition of cardano-wallet-benchmarks intentional?
> Asking as it seems to be unrelated to the PR title and description.

It's unrelated, but should be done — perhaps in a different PR.

I think that `cardano-wallet-read` should also be added.

The issue is that `nix/project-package-list.nix` influences which tests are run. I don't quite understand the inner workings of `./nix/regenerate.sh`, and I'm not sure why `cardano-wallet-read` was not added. 🤔